index.js

```diff
@@ -19,16 +19,28 @@ express()
   .set('view engine', 'ejs')
   .get('/', jsonParser, async (req, res) => {
     try {
+      if (req.query.expirysort){
+        const client = await pool.connect();
+      const result = await client.query(`SELECT p.prod_name, f.exp_dt, f.qty, p.type
+  FROM fridge_products f
+  INNER JOIN products p ON f.prod_ID = p.prod_ID
+  WHERE user_ID = '1'
+  ORDER BY exp_dt ${req.query.expirysort}`)
```

Comment:
Due to changes that we've just made to the layout of the product table, whereas this query before was perfectly fine, it now requires just one more tiny fit of specificity, in that the user_ID = '1' should be amended to f.user_ID = '1', since there is now also a user_ID column in the products table and the query currently doesn't know which table the where statement is referring to.